geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/utilities/ProtobufRequestUtilities.java

```diff
@@ -43,6 +43,18 @@
     return ClientProtocol.Message.newBuilder().setGetRequest(getRequest).build();
   }
 
+  /**
+   * Creates a request object containing a RegionAPI.ClearRequest
+   *
+   * @param regionName - Name of the region being cleared
+   * @return Request object containing the passed params.
+   */
+  public static ClientProtocol.Message createClearRequest(String regionName) {
```

Comment:
This method seems to be belong in a test utilities class...that might also be true for the rest of this file.

Reply:
I have moved it into the test directories.